.github/workflows/ci.yml

```diff
@@ -54,13 +84,19 @@ jobs:
         run: yarn test
         shell: bash
       - name: Prebuild (x64)
-        run: npm run prebuild-napi-x64
-      - name: Prebuild (arm64)
-        run: npm run prebuild-napi-arm64
-        if: ${{ matrix.os != 'ubuntu-18.04' }}
+        run: yarn prebuild-napi-x64
+      - name: Prebuild (arm64) on not Ubuntu
+        if: matrix.friendlyName != 'Ubuntu'
+        run: yarn prebuild-napi-arm64
+      - name: Prebuild (arm64) on Ubuntu
+        if: matrix.friendlyName == 'Ubuntu'
+        run: CC=aarch64-linux-gnu-gcc LINK=aarch64-linux-gnu-gcc STRIP=aarch64-linux-gnu-strip yarn prebuild-napi-arm64
+      - name: Prebuild (arm) on Ubuntu
+        if: matrix.friendlyName == 'Ubuntu'
+        run: CC=arm-linux-gnueabihf-gcc LINK=arm-linux-gnueabihf-gcc STRIP=arm-linux-gnueabihf-strip yarn prebuild-napi-arm
```

Comment:
as noted elsewhere, `node-gyp` is used here for compilation and doesn't support cross compilation on linux. so we override CC/LINK/STRIP with the appropriate cross compilers ourselves